package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "svelte-micromodal",
   "description": "Svelte wrapper around Micromodal to create light, accessible modal dialogs",
-  "version": "0.0.8",
+  "version": "0.0.9",
```

Comment:
@robinmetral  [you mentioned](https://github.com/robinmetral/svelte-micromodal/issues/6#issuecomment-1948319840) a patch version bump with a `BREAKING CHANGE` mention.

I have not mentioned `BREAKING CHANGE` in any commits as I don't know how this plays with your release process and I didn't want to jump the gun in any way.

Reply:
My bad, I'm mixing projects up :sweat_smile: this will be a plain old manual release